Add framework files

WadeBarnes opened the pull request:
- Copied from https://github.com/bcgov/orgbook-configurations

Signed-off-by: Wade Barnes <wade.barnes@shaw.ca>

WadeBarnes committed:
Add framework files

- Copied from https://github.com/bcgov/orgbook-configurations

Signed-off-by: Wade Barnes <wade.barnes@shaw.ca>

WadeBarnes committed:
Add backup build/deploy pipeline.

Signed-off-by: Wade Barnes <wade.barnes@shaw.ca>

WadeBarnes committed:
Add wallet build/deploy pipeline.

Signed-off-by: Wade Barnes <wade.barnes@shaw.ca>

WadeBarnes committed:
Add schema-spy-oracle build/deploy pipeline.

Signed-off-by: Wade Barnes <wade.barnes@shaw.ca>

WadeBarnes committed:
Add postgresql-oracle-fdw build/deploy pipeline.

Signed-off-by: Wade Barnes <wade.barnes@shaw.ca>

WadeBarnes committed:
Add db build/deploy pipelines.

Signed-off-by: Wade Barnes <wade.barnes@shaw.ca>

WadeBarnes committed:
Update formatting

Signed-off-by: Wade Barnes <wade.barnes@shaw.ca>

WadeBarnes committed:
Add mara build/deploy pipelines.

Signed-off-by: Wade Barnes <wade.barnes@shaw.ca>

WadeBarnes committed:
Rename `postgresql-oracle-fdw` to `bc-reg-fdw`

Signed-off-by: Wade Barnes <wade.barnes@shaw.ca>

WadeBarnes committed:
Add agent build/deploy pipelines.

Signed-off-by: Wade Barnes <wade.barnes@shaw.ca>

WadeBarnes committed:
Rename resources to reflect their role

- Rename resource to reflect their role, rather than their implementation.

Signed-off-by: Wade Barnes <wade.barnes@shaw.ca>

WadeBarnes committed:
Update parameters

Signed-off-by: Wade Barnes <wade.barnes@shaw.ca>

WadeBarnes committed:
Fix db-build context directory.

Signed-off-by: Wade Barnes <wade.barnes@shaw.ca>

WadeBarnes committed:
Update bc-reg-fdw-build

- to work with changes to the source repository.

Signed-off-by: Wade Barnes <wade.barnes@shaw.ca>

WadeBarnes committed:
Updates to the bc-reg-fdw deployment configuration

Signed-off-by: Wade Barnes <wade.barnes@shaw.ca>

WadeBarnes committed:
Refactor bc-reg-fdw and schema-spy-oracle deployment configurations.

- Collect and store all of the foreign database configuration elements in the bc-reg-fdw deployment resources.
- Source that same configuration information from the schema-spy-oracle deployment configuration.

Signed-off-by: Wade Barnes <wade.barnes@shaw.ca>

WadeBarnes committed:
Fix event-processor selectors

Signed-off-by: Wade Barnes <wade.barnes@shaw.ca>

WadeBarnes committed:
Add bcreg-controller build/deploy configurations.

Signed-off-by: Wade Barnes <wade.barnes@shaw.ca>

WadeBarnes committed:
Update configurations

Signed-off-by: Wade Barnes <wade.barnes@shaw.ca>

WadeBarnes committed:
Update bc-reg-fdw configuration

- Disable unused extensions.

Signed-off-by: Wade Barnes <wade.barnes@shaw.ca>

WadeBarnes committed:
Update bcreg-controller configuration with agent endpoint.

Signed-off-by: Wade Barnes <wade.barnes@shaw.ca>

WadeBarnes committed:
Fix event-processor-deploy configuration.

Signed-off-by: Wade Barnes <wade.barnes@shaw.ca>

WadeBarnes committed:
Update PVCs to support a full UAT data load.

Signed-off-by: Wade Barnes <wade.barnes@shaw.ca>

WadeBarnes committed:
Remove unused environment variables.

Signed-off-by: Wade Barnes <wade.barnes@shaw.ca>

WadeBarnes committed:
Update description for logging levels

Signed-off-by: Wade Barnes <wade.barnes@shaw.ca>

WadeBarnes committed:
Fix bcreg-controller deployment configuration.

Signed-off-by: Wade Barnes <wade.barnes@shaw.ca>

WadeBarnes committed:
Update agent-deploy configuration.

Signed-off-by: Wade Barnes <wade.barnes@shaw.ca>

WadeBarnes committed:
Remove unused controller port.

Signed-off-by: Wade Barnes <wade.barnes@shaw.ca>

WadeBarnes committed:
Update agent configuration.

Signed-off-by: Wade Barnes <wade.barnes@shaw.ca>

WadeBarnes committed:
Cleanup agent deployment configuration.

Signed-off-by: Wade Barnes <wade.barnes@shaw.ca>

WadeBarnes committed:
Fix agent deployment configuration.

Signed-off-by: Wade Barnes <wade.barnes@shaw.ca>

WadeBarnes committed:
Update event-processor deployment configuration.

Signed-off-by: Wade Barnes <wade.barnes@shaw.ca>

WadeBarnes committed:
Limit replicas to 1 until we get the communications fully sorted.

Signed-off-by: Wade Barnes <wade.barnes@shaw.ca>

WadeBarnes committed:
Add manage script.

Signed-off-by: Wade Barnes <wade.barnes@shaw.ca>

WadeBarnes committed:
Enable database management calls.

Signed-off-by: Wade Barnes <wade.barnes@shaw.ca>

esune commented on vars/sparseCheckout.groovy:
```diff
@@ -0,0 +1,20 @@
+void call(String repositoryUrl, String branch, String workingDirectory, String contextDirectory) {
```
We don't seem to be using this file in the pipeline: if it is unused I would remove it and plug it back later IF we need it.

WadeBarnes replied:
I have not started on the Jenkins Pipeline yet.

WadeBarnes committed:
Update wallet storage.

New aca-py agent implementation requires more storage.  Still need to determine how much.

Signed-off-by: Wade Barnes <wade.barnes@shaw.ca>

WadeBarnes committed:
Add getDbDiskUsage command to manage script.

Signed-off-by: Wade Barnes <wade.barnes@shaw.ca>

WadeBarnes committed:
Adjust memory and CPU resources.

Signed-off-by: Wade Barnes <wade.barnes@shaw.ca>

WadeBarnes committed:
Add getCredentialPostRate to the manage script.

Signed-off-by: Wade Barnes <wade.barnes@shaw.ca>

WadeBarnes committed:
Add listDatabases and getRecordCounts functions

Signed-off-by: Wade Barnes <wade.barnes@shaw.ca>

WadeBarnes committed:
Remove unused environment variables from bcreg-controller

Signed-off-by: Wade Barnes <wade.barnes@shaw.ca>
